src/openforms/registrations/contrib/stuf_zds/tests/test_backend.py

```diff
@@ -3818,6 +3852,1329 @@ def test_only_relevant_variables_are_taken_into_account(self):
+            {
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.bsn']": "999993677",
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.firstNames']": "Truus",
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.initials']": "T.",
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.affixes']": "van der",
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.lastName']": "Wiel",
+                "//stuf:extraElementen/stuf:extraElement[@naam='childrenKey.0.dateOfBirth']": "1960-09-12",
+            },
+        )
+
+        # make sure that nothing is registered as zaakbetrokkene
+        self.assertXPathNotExists(
+            xml_doc,
+            "//zkn:object/zkn:heeftAlsOverigBetrokkene[@stuf:entiteittype='ZAKBTROVR']/zkn:gerelateerde/zkn:natuurlijkPersoon/bg:inp.bsn",
+        )
+
+    def test_create_zaak_with_two_form_steps_and_children_as_betrokkene(self):
+        """
+        Input
+
+        - Form step 1: children component
+        - Form step 2: editgrid component
+        - Children data: prefilled
+        - Registration type: betrokkene for children and extraElementen for editgrid
+        - Children selection: disabled
+        """
+
+        form = FormFactory.create()
+        step1 = FormStepFactory.create(
+            form=form,
+            form_definition__configuration={
+                "components": [
+                    {
+                        "type": "children",
+                        "key": "children",
+                        "label": "Children",
+                        "enableSelection": False,
+                        "registration": {"attribute": RegistrationAttribute.children},
+                    },
+                ]
+            },
+        )
+        step2 = FormStepFactory.create(
+            form=form,
+            form_definition__configuration={
+                "components": [
+                    {
+                        "type": "editgrid",
+                        "key": "extraChildDetails",
+                        "components": [
+                            {"type": "bsn", "key": "bsn", "label": "BSN"},
+                            {
+                                "type": "textfield",
+                                "key": "childName",
+                                "label": "Child name",
+                            },
+                        ],
+                    }
+                ]
+            },
+        )
+        FormVariableFactory.create(
+            key="children_immutable",
+            form=form,
+            user_defined=True,
+            prefill_plugin="family_members",
+            prefill_options={
+                "type": "children",
+                "mutable_data_form_variable": "children",
+                "min_age": None,
+                "max_age": None,
+            },
+        )
+
+        FormLogicFactory.create(
```

Comment:
shouldn't this have been deleted according to earlier PR feedback? I also notice that the logic action type now has an outdated value `synchronize-children`, so it's not representative now anyway. Typically for these kind of things you really want to use the constants/enums defined instead of the string literals, for exactly this reason :)